frontend/src/v5/ui/components/shared/breadcrumbs/breadcrumbs.styles.ts

```diff
@@ -26,19 +26,27 @@ export const Container = styled(Breadcrumbs)`
 	}
 `;
 
+const BreadcrumbTabbedStyle = css`
+	&:hover {
+		text-decoration: none;
+	}
+
+	&.MuiButton-text.Mui-focusVisible {
+		background-color: ${({ theme }) => theme.palette.secondary.mid};
+	}
+`;
+
 export const InteractiveBreadcrumb = styled(Button).attrs({
```

Comment:
instead of having  `BreadcrumbTabbedStyle` being repeated, we can make InteractiveBreadcrumb  a styled Breadcrumb and just add the typograhy and the color. I tested in my computer and it seems to look the same as these changes unless I'm missing something.

```suggestion
export const InteractiveBreadcrumb = styled(Breadcrumb)`
```

Reply:
Fixed